src/test/fuzz/net.cpp

```diff
@@ -0,0 +1,164 @@
+        }
+        case 4: {
+            node.SetRecvVersion(fuzzed_data_provider.ConsumeIntegral<int>());
+            break;
+        }
+        case 5: {
+            const CNode* add_ref_node = node.AddRef();
+            assert(add_ref_node == &node);
+            break;
+        }
+        case 6: {
+            if (node.GetRefCount() > 0) {
+                node.Release();
+            }
+            break;
+        }
+        case 7: {
+            if (node.m_addr_known == nullptr) {
+                break;
+            }
+            const std::optional<CAddress> addr_opt = ConsumeDeserializable<CAddress>(fuzzed_data_provider);
+            if (!addr_opt) {
+                break;
+            }
+            node.AddAddressKnown(*addr_opt);
+            break;
+        }
+        case 8: {
+            if (node.m_addr_known == nullptr) {
+                break;
+            }
+            const std::optional<CAddress> addr_opt = ConsumeDeserializable<CAddress>(fuzzed_data_provider);
+            if (!addr_opt) {
+                break;
+            }
+            FastRandomContext fast_random_context{ConsumeUInt256(fuzzed_data_provider)};
+            node.PushAddress(*addr_opt, fast_random_context);
+            break;
+        }
+        case 9: {
+            const std::optional<CInv> inv_opt = ConsumeDeserializable<CInv>(fuzzed_data_provider);
+            if (!inv_opt) {
+                break;
+            }
+            node.AddInventoryKnown(*inv_opt);
+            break;
+        }
+        case 10: {
+            const std::optional<CInv> inv_opt = ConsumeDeserializable<CInv>(fuzzed_data_provider);
+            if (!inv_opt) {
+                break;
+            }
+            node.PushInventory(*inv_opt);
+            break;
+        }
+        case 11: {
+            node.PushBlockHash(ConsumeUInt256(fuzzed_data_provider));
+            break;
+        }
+        case 12: {
+            const std::optional<CService> service_opt = ConsumeDeserializable<CService>(fuzzed_data_provider);
+            if (!service_opt) {
+                break;
+            }
+            node.SetAddrLocal(*service_opt);
+            break;
+        }
+        case 13: {
+            const std::vector<uint8_t> b = ConsumeRandomLengthByteVector(fuzzed_data_provider);
+            bool complete;
+            node.ReceiveMsgBytes((const char*)b.data(), b.size(), complete);
+            break;
+        }
+        }
+    }
+
+    (void)node.GetAddrLocal();
+    (void)node.GetAddrName();
+    (void)node.GetId();
+    (void)node.GetLocalNonce();
+    (void)node.GetLocalServices();
+    (void)node.GetMyStartingHeight();
+    (void)node.GetRecvVersion();
+    const int ref_count = node.GetRefCount();
+    assert(ref_count >= 0);
+    (void)node.GetSendVersion();
+    (void)node.IsAddrRelayPeer();
+
+    const NetPermissionFlags net_permission_flags = fuzzed_data_provider.ConsumeBool() ? fuzzed_data_provider.PickValueInArray<NetPermissionFlags>({
```

Comment:
```suggestion
    const NetPermissionFlags net_permission_flags = fuzzed_data_provider.ConsumeBool() ?
    fuzzed_data_provider.PickValueInArray<NetPermissionFlags>({
```

Couldn't this start a new line after the `?` Seems overly wasteful to fill the screen with 8 * 90 pure whitespace

Reply:
Sure! Now fixed: `clang-format`:ed version replaced with another `clang-format`:ed version after manual re-arrangement.

When it comes to formatting I really don't have any personal preferences: I'm happy as long as `clang-format` is happy :)